.github/workflows/fast.yml

```diff
@@ -47,30 +47,57 @@ jobs:
         RUSTDOCFLAGS: "-D warnings"
 
   Tests:
+    name: "Tests (os: ${{ matrix.os }}, flags: \"${{ matrix.flags }}\") (partition: ${{ matrix.partition }}/2)"
     runs-on: ${{ matrix.os }}
     timeout-minutes: 60
     strategy:
       matrix:
         include:
           - os: ubuntu-latest
             flags: "--features commonware-runtime/iouring-storage"
+            partition: 1
```

Comment:
Is there a method to the madness here on applying partition IDs?

Reply:
Seems error-prone to have to list all partitions out like this (can we not make partitions an array or something)?

Reply:
I was going to say I don't think so, but turns out `exclude` actually works here (I remember having trouble with it in the past.)

Updated; Good bit more clean imo.